lib/utilities/reminder_banner.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:flutter/material.dart';
+
+import 'package:mm_flutter_app/themes/palette.dart';
+
+class ReminderBanner {
+  static void showReminderBanner(
+      BuildContext context, Icon? icon, String text, Function() onOpenPressed) {
+    ScaffoldMessenger.of(context).showMaterialBanner(
+      MaterialBanner(
+        padding: const EdgeInsets.all(16),
+        content: Text(text),
+        leading: icon,
+        backgroundColor: Colors.white70,
```

Comment:
Totally optional: what do you think about abstracting the colors in a Theme object? Then we can easily changed that later on without having to change in the widget.

Reply:
Yes absolutely! Using an App level Theme will make a lot of these declarations simpler. At this stage I am just quickly putting stuff together to see what works and what doesn't. Once we see the patterns in colors and styles they should be abstracted into constants / themes. I can take care of doing this next